cli/gencert/gencert.go

```diff
@@ -83,7 +85,19 @@ func gencertMain(args []string, c cli.Config) error {
 			}
 
 		}
-		cli.PrintCert(key, csrPEM, cert)
+		if c.OutFile == "pkcs12" {
+			password := make([]byte, 0)
+			if c.Password != "0" {
+				password = []byte(c.Password)
+			}
+			pkcs12Key, err := pkcs12.ParseAndEncode(key, cert, password)
+			if err != nil {
+				log.Errorf("%v\n", err)
+			}
+			cli.PrintCertAndFile(key, csrPEM, cert, pkcs12Key)
```

Comment:
An exported PKCS12 key is binary data, so you might not want to dump it to standard output.